exercises/binary-search/instructions.md

```diff
@@ -5,17 +5,17 @@ Your task is to implement a binary search algorithm.
 A binary search algorithm finds an item in a list by repeatedly splitting it in half, only keeping the half which contains the item we're looking for.
 It allows us to quickly narrow down the possible locations of our item until we find it, or until we've eliminated all possible locations.
 
-```exercism/caution
+````exercism/caution
 Binary search only works when a list has been sorted.
-```
+````
```

Comment:
It's either three backticks, or four tildes, according to the styleguide. (See the section called **Special blocks** here). https://exercism.org/docs/building/markdown/markdown

Reply:
That section says 4 tildes unless in exceptional circumstances, which this doesn't seem to be?

Reply:
Reading is very hard, okay.

Reply:
```suggestion
~~~~
```

Reply:
@SaschaMann I'm not sure this was clear, but I was agreeing with you 😄 

I've submitted a PR that fixes this in main, so once that is merged we can rebase this branch against that one.